docs/source/Coding/Setting-up-PyCharm.md

```diff
@@ -1,57 +1,104 @@
 1. Go to `Run > Edit Configutations...`
-1. Click the plus-symbol to add a new configuration and choose Python
-1. Add the script: `\<yourrepo\>\evenv\Scripts\evennia_launcher.py` (substitute your virtualenv if it's not named `evenv`)
-1. Set script parameters to: `start -l` (-l enables console logging)
-1. Ensure the chosen interpreter is from your virtualenv
-1. Set Working directory to your `mygame` folder (not evenv nor evennia)
-1. You can refer to the PyCharm documentation for general info, but you'll want to set at least a config name (like "MyMUD start" or similar).
+2. Click the plus-symbol to add a new configuration and choose Python
+3. Add the script: `\<yourprojectfolder>\.venv\Scripts\evennia_launcher.py` (substitute your virtualenv if it's not named `evenv`)
+4. Set script parameters to: `start -l` (-l enables console logging)
+5. Ensure the chosen interpreter is your virtualenv
+6. Set Working directory to your `mygame` folder (not your project folder nor evennia)
+7. You can refer to the PyCharm documentation for general info, but you'll want to set at least a config name (like "MyMUD start" or similar).
 
-Now set up a "stop" configuration by following the same steps as above, but set your Script parameters to: stop (and name the configuration appropriately).
+A dropdown box holding your new configurations should appear next to your PyCharm run button. 
+Select it start and press the debug icon to begin debugging.
 
-A dropdown box holding your new configurations should appear next to your PyCharm run button. Select MyMUD start and press the debug icon to begin debugging.  Depending on how far you let the program run, you may need to run your "MyMUD stop" config to actually stop the server, before you'll be able start it again.
+#### On Linux
+1. Go to `Run > Edit Configutations...`
+2. Click the plus-symbol to add a new configuration and choose Python
+3. Add the script: `/<yourprojectfolder>/.venv/bin/twistd` (substitute your virtualenv if it's not named `evenv`)
+4. Set script parameters to: `--python=/<yourprojectfolder>/.venv/lib/python3.11/site-packages/evennia/server/server.py --logger=evennia.utils.logger.GetServerLogObserver --pidfile=/<yourprojectfolder>/<yourgamefolder>/server/server.pid --nodaemon`
+5. Add an environment variable `DJANGO_SETTINGS_MODULE=server.conf.settings`
+6. Ensure the chosen interpreter is your virtualenv
+7. Set Working directory to your game folder (not your project folder nor evennia)
+8. You can refer to the PyCharm documentation for general info, but you'll want to set at least a config name (like "MyMUD start" or similar).
+
+A dropdown box holding your new configurations should appear next to your PyCharm run button. 
+Select it start and press the debug icon to begin debugging.
+Note that this only starts the server process, you will still need to start the portal
+using evennia start AFTER launching the server process.
 
 ### Alternative config - utilizing logfiles as source of data
```

Comment:
This is an old part of the tutorial, is it still valid? Some of the images are not very clear to me, and also shows Python 2.7 ...

Reply:
The alternative config?
Not tested, will try it.